coins/src/adapters/tokenMapping.json

```diff
@@ -265,6 +265,11 @@
       "decimals": "18",
       "symbol": "rcbETH",
       "to": "coingecko#ethereum"
+    },
+    "0xcc7ff230365bd730ee4b352cc2492cedac49383e": {
+      "decimals": "18",
+      "symbol": "hyUSD",
+      "to": "coingecko#high-yield-usd-base"
```

Comment:
this is from coingecko terminal, we cant price from that
need to be from coingecko it self

Reply:
This asset is also on Coingecko itself https://www.coingecko.com/en/coins/high-yield-usd-base

If this approach isn't correct for getting pricing for this asset, could you please point me to the proper way?